frontend/packages/ux-editor/src/components/TextResource/TextResource.tsx

```diff
@@ -47,6 +55,12 @@ export const TextResource = ({
     setIsOpen(true);
   };
 
+  useEffect(() => {
+    if (formItemId !== prevFormItemId || formLayoutName !== prevFormLayoutName) {
```

Comment:
Nice catch also checking for `formLayoutName`. I didn't think that was necessary until I tested some myself! 😄

Reply:
Yeah, almost missed it! Thanks for the review! 😄